index.template.html

```diff
@@ -375,15 +375,15 @@
   externally when a <a>Thing</a> has resource restrictions (e.g., limited memory space)
   or when a Web of Things-compatible legacy device is retrofitted 
   with a Thing Description.
-  Furthermore, this document introduces the Thing Model, which allows to describe 
-  only the model or class of an IoT entity. Thing Models can be seen as a template for Thing 
+  Furthermore, this document introduces the Thing Model, which allows authors to describe
+  only the model or class of an Internet of Thing (IoT) entity. Thing Models can be seen as a template for Thing 
   Description instances, but with reduced constraints such as no or few requirements for specific 
   communication metadata. 
   </p>
 
   <p>
     This specification describes a superset of the features defined in Thing Description 1.0 [[WOT-THING-DESCRIPTION]]. 
-    Unless otherwise specified, documents created with version 1.0 of this specification remain 
+    Unless otherwise specified, documents created with terms from the information model version 1.0 of this specification remain 
```

Comment:
Where can a reader find these incompatibilities?

Reply:
It is unusual to have such a link in the abstract. The once relevant place is section 5.3.1 so far. Also see #1334